### DISCUSSION TITLE
How to show SolidJS examples?

### DISCUSSION THREAD

**Discussion by QWu4xYV** [cschroeter/park-ui#219] [Q&A] [answered]
In the menu that pops down when clicking on "Show code", the "Solid" (and "Vue") tabs are disabled. Is that because:

1. There aren't usage examples for those languages yet, but they're planned (I didn't see any tracking issue for this)?
2. There are usage examples, but they can't be rendered because the docs site default display is written in React in a way that doesn't support that?

It's not too difficult to tweak the React examples, but having official Solid examples would be helpful in case of debugging any tweaks I would make.

**Comment by cschroeter** [comment, marked as answer]
@QWu4xYV 

Thanks for taking your time to start this discussion. Every JS and CSS Framework has its own storybook. If you want to see the components in Solid and Panda:

1. Clone the project
2. Install the deps using bun `bun i`
3. Start storybook `bun panda:solid storybook`

**Reply by QWu4xYV** [reply]
Thanks so much! I hoped and figured it was something like that... In that case, I think it would be a good idea for those disabled tabs in the current documentation to link to this information, so other people who are looking for the examples can easily find them. 

Regardless, thank you so much for this great library! I'm enjoying it a lot!

**Reply by cschroeter** [reply]
@QWu4xYV 

Thanks for your nice feedback. We will improve the docs as we go. Thanks for your suggestion.
